rskj-core/src/main/java/co/rsk/config/BridgeConstants.java

```diff
@@ -37,6 +37,7 @@
     protected int maxBtcHeadersPerRskBlock;
 
     protected Coin minimumLockTxValue;
+    protected Coin minimumLockTxValueAfterIris;
```

Comment:
Why lock and not pegin?

Reply:
keeping the original criteria, but I like the idea of change.